<!-- -*- mode: XML -*- -->
<!-- run web can dumper for can message from mobile eye camera as example -->

<launch>

  <!-- node for can dumper -->
  <node pkg="can_dumper" type="can_dumper_node" name="can_dumper_node" output="screen">
    <param name="collect_interval" value="5" />
    <param name="broadcast_time" value="50" />
    <param name="device_config_file" value="$(find can_dumper)/config/device.conf" />
  </node>

  <node pkg="rosbag" type="record" name="can_record"
    args="record -o /home/master/test_jinbo/data/
                    /can_dumper_node/can_data
                    /can_dumper_node/can_visualization"/>

</launch>
